recipes/openllm-playground/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "openllm-playground" %}
+{% set version = "0.4.31" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openllm-{{ version }}.tar.gz
+  sha256: cd2850a6c46b8f801325265a8b7e91ee193a8acfb848deccb7efc6273ee91895
+
+build:
+  skip: true  # [win or osx]
+  skip: true  # [py<38]
+  entry_points:
+    - openllm = openllm_cli.entrypoint:cli
+    - openllm-dive-bentos = openllm_cli.extension.dive_bentos:cli
+    - openllm-get-containerfile = openllm_cli.extension.get_containerfile:cli
+    - openllm-get-prompt = openllm_cli.extension.get_prompt:cli
+    - openllm-list-bentos = openllm_cli.extension.list_bentos:cli
+    - openllm-list-models = openllm_cli.extension.list_models:cli
+    - openllm-playground = openllm_cli.extension.playground:cli
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - hatchling ==1.18.0
+    - hatch-vcs ==0.3.0
+    - hatch-fancy-pypi-readme ==23.1.0
+    - pip
```

Otherwise they will clobber eachother.
```suggestion
    - pip
    - openllm {{ version }}
```